x-pack/legacy/plugins/apm/cypress/cypress/support/step_definitions/apm.ts

```diff
@@ -0,0 +1,48 @@
+/// <reference types="Cypress" />
+
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { loginAndWaitForPage } from '../../integration/helpers';
+
+const { Given, When, Then } = require('cypress-cucumber-preprocessor/steps');
+
+Given(`a user browses the APM UI application`, () => {
```

Comment:
There is no difference using `Given('foo')`, `When('foo')`, `Then('foo')`... They technically could be defined in any format. **BUT** it's important to use the proper word for the sake of semantics! We are providing information to an occasional reader so he/she is able to properly understand the specification/behavior